builder/README.md

```diff
@@ -1,6 +1,9 @@
 # Deis Builder
 
-A Docker image that builds Docker images, for use in the [Deis](http://deis.io/) open source PaaS.
+**WARNING:** This is the experimental Go-based builder.
```

Comment:
Should we remove this? After the merge, this will indeed be the official builder :)